lib/inferno/apps/cli/templates/spec/%lib_name%/patient_group_spec.rb.tt

```diff
@@ -0,0 +1,105 @@
+RSpec.describe <%= module_name %>::PatientGroup do
```

Comment:
`%lib_name%` in the path isn't being updated:
```
ᐅ git status
...
Untracked files:
  ...
	spec/%lib_name%/
```

Reply:
fixed